simulationworkflowschema/molecular_dynamics.py

```diff
@@ -821,6 +992,90 @@ def normalize(self, archive, logger):
             self.value = self._rg_results.get("value")
 
 
+class FreeEnergyCalculations(TrajectoryProperty):
+    """
+    Section containing information regarding the instantaneous (i.e., for a single configuration)
+    values of free energies calculated via thermodynamic perturbation.
+    The values stored are actually infinitesimal changes in the free energy, determined as derivatives
+    of the Hamiltonian with respect to the coupling parameter (lambda) defining each state for the perturbation.
+    """
+
+    m_def = Section(validate=False)
+
+    method_ref = Quantity(
+        type=Reference(FreeEnergyCalculationParameters.m_def),
+        shape=[],
+        description="""
+        Links the free energy results with the method parameters.
+        """,
+    )
+
+    lambda_index = Quantity(
+        type=int,
+        shape=[],
+        description="""
+        Index of the lambda state for the present simulation within the free energy calculation workflow.
+        I.e., lambda = method_ref.lambdas.values[lambda_index]
+        """,
+    )
+
+    n_states = Quantity(
+        type=int,
+        shape=[],
+        description="""
+        Number of states defined for the interpolation of the system, as indicate in `method_ref`.
+        """,
+    )
+
+    value_unit = Quantity(
```

Comment:
I think that if you elevate this to `BaseProperty`, you can also define the description as in here, **or**, better define each value units separately? Tho I admit is more annoying.